### ISSUE TITLE
Documents addedBy value is inconsistent

### ISSUE THREAD

**Issue by abrahamseni** [reapit/foundations#11170]
Related to #11148 

The `addedBy` property value has some bugs:
1. it can't be `null`.
2. if we upload it from API, does it come from loginIdentity.name? because it won't show in AC. i.e: 
```js
// upload from API
"addedBy": "Matt Goddard"

// upload from AC
"addedBy": "MTGD"
```

I think we should use a name abbreviation for consistency with AC, but I don't know where to get it, is there any rule to get that name abbreviation?

**Comment by RWilcox-Reapit** [comment]
Hi @abrahamseni,

The `addedBy` field set by the platform API's is indeed the calling users name OR the calling app name if there is no user in context (Client Credentials flow)

Is the `addedBy` for documents uploaded via platform now showing for you?
If so please could you provide us with the customer and documents identifier in question and we can look into it.

Thanks

**Comment by abrahamseni** [comment]
> Hi @abrahamseni,
> 
> The `addedBy` field set by the platform API's is indeed the calling users name OR the calling app name if there is no user in context (Client Credentials flow)
> 
> Is the `addedBy` for documents uploaded via platform now showing for you? If so please could you provide us with the customer and documents identifier in question and we can look into it.
> 
> Thanks

@RWilcox-Reapit,
Yes I could see `addedBy` property and upload the documents fine.
The problem is when I upload it, the `addedBy` value is from `loginIdentity` such as `Matt Goddard`, this won't show in AC, because I think AC ignores it and reads differently.

screenshot:
the blue mark is when uploaded from AC, the `addedBy` value is `MTGD` not `Matt Goddard`

![Screenshot 2024-05-31 021120](https://github.com/reapit/foundations/assets/94949137/6f3117ae-d65d-4be2-bc98-110b0b2d66fc)

documents: 
env: SAV, PL == Platform, AC == Agency Cloud
![Screenshot 2024-05-31 023451](https://github.com/reapit/foundations/assets/94949137/e61bf695-0ced-4f6d-b676-5cc7f44236d0)

**Comment by RWilcox-Reapit** [comment]
Hi @abrahamseni ,

Thanks for bringing this to our attention, a fix is currently waiting to be merged for this.

Ryan

**Comment by RWilcox-Reapit** [comment]
Hi @abrahamseni 

The fix is in now and the values align with AC. 
`AddedBy` will now always be the identifier of the negotiator who created the document.

Thanks

**Comment by abrahamseni** [comment]
@RWilcox-Reapit 
Thanks, it works.
